src/app/shared/footer/footer.component.html

```diff
@@ -0,0 +1,12 @@
+<mat-divider></mat-divider>
+<footer class="pt-10">
+  <div fxLayout="row" fxLayoutAlign="space-between center">
+    <ul fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+      <li>CREATIVE TIM</li>
+      <li>ABOUT US</li>
+      <li>BLOG</li>
+      <li>LICENSES</li>
+    </ul>
+    <p> © 2022, made with love by Creative Tim for a better web.</p>
```

Comment:
don't use symbol directly rather used code like &copy;



<br class="Apple-interchange-newline">

Reply:
change the styling also using the flex layouting